src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HearingsHelper.java

```diff
@@ -87,4 +91,32 @@ public static void updatePostponedDate(CaseData caseData) {
         }
 
     }
+
+    public static List<String> hearingTimeValidation(CaseData caseData) {
```

Comment:
Too many identations. You could simplify this

Reply:
Refactored